samples/nextjs/src/components/Layout.js

```diff
@@ -0,0 +1,66 @@
+import Head from 'next/head';
+import Link from 'next/link';
+import { useI18n } from 'next-localization';
+import { Placeholder, withSitecoreContext, VisitorIdentification } from '@sitecore-jss/sitecore-jss-nextjs';
+
+// This is boilerplate navigation for sample purposes. Most apps should throw this away and use their own navigation implementation.
+// Most apps may also wish to use GraphQL for their navigation construction; this sample does not simply to support disconnected mode.
+const Navigation = () => {
+  const i18n = useI18n();
+
+  return (
+    <div className="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom">
+      <h5 className="my-0 mr-md-auto font-weight-normal">
+        <a className="text-dark" href="/"><img src="/sc_logo.svg" alt="Sitecore" /></a>
```

Comment:
@ambrauer Is `next/Image` working here?

Reply:
Good call, I hadn't really looked into using that since it's so hot off the press.